roles/nf-core/templates/site.config.j2

```diff
@@ -30,6 +30,14 @@ process {
         time = { 2.d * task.attempt }
   }
 
+
+/* Temporary downgrade of bismark to work around the issue described here:
+   https://github.com/FelixKrueger/Bismark/issues/652#issuecomment-2750781652
+   This workaround should be re-evaluated when upgrading the methylseq pipeline */
```

Comment:
Maybe add a comment in the following file to remember this next time we update this pipeline:
 https://github.com/NationalGenomicsInfrastructure/miarka-provision/blob/5a564e21491b132d01517ddda81407909a6bf701/roles/nf-core/defaults/main.yml#L37

Having `TODO` or `FIXME` in the comment also helps to grep for such issues, see `pipeline.yml` for instance: https://github.com/NationalGenomicsInfrastructure/miarka-provision/blob/5a564e21491b132d01517ddda81407909a6bf701/roles/nf-core/tasks/pipeline.yml#L52